babel-code-frame/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
you do not need this. other `@types` packages do not take a dependency on js packages.